gulpfile.js

```diff
@@ -57,7 +58,12 @@ gulp.task('tracks', function () {
 
 gulp.task('copy', ['resources', 'tracks'])
 // build everything once, probably for production
-gulp.task('once', ['browser', 'node', 'copy'])
+gulp.task('once', ['browser', 'node', 'copy'], function () {
+  return gulp.src('app/chrome/**')
+             .pipe(zip('chrome.zip'))
```

Comment:
@dfarache please add this file to the gitignore if not already there

Reply:
@carocad do you mean the chrome.zip? it is already ignored

Reply:
ok cool, just checking :)